src/Chapter1.hs

```diff
@@ -620,8 +645,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = ten + hundred
+    where
+        last2 = mod n 100
+        ten = mod last2 10
+        hundred = div last2 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(hundred, ten) = divMod last2 10
```

You can see how we could pattern match on the pair 🙂 

Also, the `last2` function you specify in here is actually the `lastDigit` function from the previous task :wink:

Reply:
Thank you. 

Made the suggested changes.